### DISCUSSION TITLE
Issues with Image Caching in Next.js: Previous Images Displayed Instead of Updated Ones

### DISCUSSION THREAD

**Discussion by Farhan-Ajmal-1** [vercel/next.js#70316] [Help]
### Summary

The problem I had was with images in Next.js. The website shows images when running locally, but they don’t appear when it's live. I fixed it before by upgrading from version 13 to 14, but I’m not sure why that worked—I just tried upgrading the version and it solved the issue

I'm facing another issue. The images are coming from Hostinger and are updated with the same name, but the website still shows the previous images. When I switch from the Next.js Image component to the Chakra UI Image component, it works perfectly fine. I don't understand why Next.js images behave this way.

**Comment by Juneezee** [comment]
Looks like a cache issue to me.

> The images are coming from Hostinger

If your images don't have a SHA-hash in the file name e.g. `img.png` instead of `img-43c09c11.png`, then Next.js could be reusing the old cached version.

You can try configuring the `minimumCacheTTL` option https://nextjs.org/docs/app/api-reference/components/image#minimumcachettl
